### PR TITLE
New package: tiny-0.8.0

### PR DESCRIPTION
**Pull request by eoli3n**
Closes https://github.com/void-linux/void-packages/issues/27180 and https://github.com/osa1/tiny/issues/286

### CODE REVIEW

File: srcpkgs/tiny-irc-client/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+build_style=cargo
+configure_args="--no-default-features --features=desktop-notifications --features=tls-native"
+hostmakedepends="pkg-config"
+makedepends="libressl-devel dbus-devel"
+short_desc="Tiny terminal IRC client"
+maintainer="eoli3n <jkirsz@gmail.com>"
+license="MIT"
+homepage="https://github.com/osa1/tiny"
+distfiles="https://github.com/osa1/tiny/archive/v${version}.tar.gz"
+wrksrc="tiny-${version}"
```
Comment (ndowens) [inline srcpkgs/tiny-irc-client/template:14]: Should be below revision
also build_wrksrc should be as well

**Review by ndowens — CHANGES_REQUESTED** [review]

File: srcpkgs/tiny-irc-client/template [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'tiny-irc-client'
+pkgname=tiny-irc-client
```
Comment (ahesford) [inline srcpkgs/tiny-irc-client/template:2]: Package name should be `tiny`, also update commit message to reflect name change.

**Review by ahesford — CHANGES_REQUESTED** [review]

File: srcpkgs/tiny-irc-client/template (hunk 2 above)
Comment (eoli3n) [reply srcpkgs/tiny-irc-client/template:2]: ok

Comment (ndowens) [reply srcpkgs/tiny-irc-client/template:2]: Once you change it to 'tiny' you wont need the wrksrc* lines

Comment (eoli3n) [reply srcpkgs/tiny-irc-client/template]: yes i noticed that, thanks, but i still need build_wrksrc right ?

File: srcpkgs/tiny-irc-client/template (hunk 1 above)
Comment (eoli3n) [reply srcpkgs/tiny-irc-client/template]: @ndowens i think i still need ``build_wrksrc`` : https://github.com/osa1/tiny/issues/286#issuecomment-748977566

File: srcpkgs/tiny/template [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'tiny'
+pkgname=tiny
+version=0.8.0
+revision=1
+build_wrksrc="tiny"
+build_style=cargo
+configure_args="--no-default-features --features=desktop-notifications --features=tls-native"
+hostmakedepends="pkg-config"
+makedepends="libressl-devel dbus-devel"
+short_desc="Tiny terminal IRC client"
```
Comment (osa1) [inline srcpkgs/tiny/template:10]: By "tiny" do you mean as an adjective and as the name of the program? If the latter I'd use "tiny" (lowercase) here0.

File: srcpkgs/tiny/template [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+maintainer="eoli3n <jkirsz@gmail.com>"
+license="MIT"
+homepage="https://github.com/osa1/tiny"
+distfiles="https://github.com/osa1/tiny/archive/v${version}.tar.gz"
+checksum=acaf0b7c3515bdfd0c80b4a7274aec44d283025c6e40508e450525167f73e447
```
Comment (Johnnynator) [inline srcpkgs/tiny/template:15]: Add a empty newline after checksum (just some informal convention to have a emtpy lines before a function)

File: srcpkgs/tiny/template (hunk 3 above)
Comment (Johnnynator) [reply srcpkgs/tiny/template:10]: We do have Descriptions always starting with a Uppercase letter, so `Tiny` is the correct one.

**Comment by eoli3n** [comment]
ping @Johnnynator @ericonr @ahesford

**Comment by ahesford** [comment]
Please rebase and make sure this builds everywhere. We have changed SSL implementations since this last ran through CI.

**Comment by eoli3n** [comment]
Thanks

**Pull request merged**